javascript/ql/src/semmle/javascript/dataflow/Nodes.qll

```diff
@@ -246,8 +246,9 @@ class GlobalVarRefNode extends DataFlow::ValueNode, DataFlow::DefaultSourceNode
  */
 DataFlow::SourceNode globalObjectRef() {
   // top-level `this`
-  exists (ThisNode globalThis | result = globalThis |
-    not exists(globalThis.getBinder())
+  exists (StmtContainer sc |
```

Comment:
This change seems unrelated to this PR.

Reply:
In a sense, yes. It is, however, necessary to prevent a negative recursion.

Reply:
:+1: